src/Popover/PopoverContent.js

```diff
@@ -55,12 +57,19 @@ type Props = {
 const arrowSize = '8px';
 
 export const componentTheme = (baseTheme: Object) => ({
+  ...popoverArrowComponentTheme(baseTheme),
```

Comment:
I think we may want this here anyway since PopoverArrow isn't used outside of PopoverContent. So this would allow someone to use PopoverContent and style the PopoverArrow within it.

Reply:
ah i understand. fixed